### PR TITLE
Orams build pipelines

### PR DESCRIPTION
**Pull request by bsigney**
Build Pipeline configuration to staging environment

### CODE REVIEW

File: backend/circle.yml [hunk 1]
```diff
@@ -44,18 +44,18 @@ deployment:
     commands:
```
Comment (matt-sm) [inline backend/circle.yml]: delete this file?

File: backend/app/api/services/__init__.py [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
 from .suppliers import SuppliersService
+from .brief_responses import BriefResponsesService
 from .users import UsersService
 
 prices = PricesService()
 audit_service = AuditService()
 audit_types = AuditTypes
 assessments = AssessmentsService()
 suppliers = SuppliersService()
+brief_responses_service = BriefResponsesService()
```
Comment (matt-sm) [inline backend/app/api/services/__init__.py]: was there a reason you had to add this back in - it's not needed for orams

File: backend/scripts/run_tests.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ex
+pep8 app
+pep8 tests
+py.test --cov=app --cov-report term-missing --cov-report=html:${CIRCLE_ARTIFACTS-.}/htmlcov
```
Comment (matt-sm) [inline backend/scripts/run_tests.sh]: i would be inclined to only run the tests in `tests/fixtures`

File: backend/app/api/services/__init__.py (hunk 2 above)
Comment (bsigney) [reply backend/app/api/services/__init__.py]: the `test_tasks.py` was failing and wasn't sure if this test was relevant or not so just put the `brief_responses_services` back on until I can look at it.

Comment (bsigney) [reply backend/app/api/services/__init__.py]: removed as per discussion

File: backend/circle.yml (hunk 1 above)
Comment (bsigney) [reply backend/circle.yml]: deleted

**Pull request merged**